ladybug_comfort/adaptive.py

```diff
@@ -80,53 +60,108 @@ def adaptive_comfort_en15251(t_prevail, ta, tr, vel, conditioning=0):
         conditioning: A number between 0 and 1 that represents how "conditioned" vs.
             "free-running" the building is.
-                0 = free-running (completely passive with no running air conditioning)
+                0 = free-running (completely passive with no air conditioning)
                 1 = conditioned (no operable windows and fully air conditioned)
-            Note that this parameter is not an official part of the EN-15251 standard.
-            For more information on how adaptive comfort methods can be applied
-            to conditioned buildings, see the neutral_temperature_conditioned function.
+        model: The comfort standard, which will be used to represent the "free-running"
+            function.  Chose from: 'EN-15251', 'ASHRAE-55'.
 
     Returns:
         result: A dictionary containing results with the following keys:
             to : Operative Temperature [C].
             t_comf : Adaptive comfort neutral temperature (desired by occupants) [C].
-            ce : Cooling effect as a result of elevated air speed [C].
             deg_comf: The difference between the operative temperature (to)
                 and the adaptive comfort neutral temperature (t_comf) [C].
                 Negative values inidcate cool conditions and positive values
                 indicate varm conditions.
     """
-    to = (ta + tr) / 2  # operative temperature
-
     # fix upper and lower outdoor temperatures if outside the range of the model
     if t_prevail < 10.0:
         t_prevail = 10.0
     elif t_prevail > 30:
         t_prevail = 30
 
     # get the neutral temperature
-    if conditioning == 0:
-        t_comf = neutral_temperature_en15251(t_prevail)
-    else:
-        t_comf = neutral_temperature_conditioned(t_prevail, conditioning, 'EN-15251')
-
-    # get the cooling effect as a result of elevated air speed
-    ce = cooling_effect_en15251(vel, to)
-
+    t_comf = neutral_temperature_conditioned(t_prevail, conditioning, model)
     result = {}
     result['to'] = to
     result['t_comf'] = t_comf
-    result['ce'] = ce
     result['deg_comf'] = to - t_comf
 
     return result
 
 
+def get_adaptive_comfort_conditioned_function(conditioning, model):
+    """Get an adaptive_comfort_conditioned function with pre-set conditioning/model.
+
+    Args:
+        conditioning: A number between 0 and 1 that represents how "conditioned" vs.
+            "free-running" the building is.
+                0 = free-running (completely passive with no air conditioning)
+                1 = conditioned (no operable windows and fully air conditioned)
+        model: The comfort standard, which will be used to represent the "free-running"
+            function.  Chose from: 'EN-15251', 'ASHRAE-55'.
+    """
+    def comfort_funct(t_prevail, to):
+        return adaptive_comfort_conditioned(t_prevail, to, conditioning, model)
+    return comfort_funct
+
+
+def get_operative_temperature(ta, tr):
```

Comment:
This creates difficulties because it will overwrite the name of the `operative_temperature` property that exists on some objects.  I can just all it `t_operative()` if you don't like the get_.